bpdm-gate-api/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/api/GateBusinessPartnerApi.kt

```diff
@@ -95,4 +93,22 @@ interface GateBusinessPartnerApi {
         @ParameterObject @Valid paginationRequest: PaginationRequest = PaginationRequest()
     ): PageDto<BusinessPartnerOutputDto>
 
+    @Operation(
+        summary = "Create or update business partners from uploaded CSV file",
+        description = "Create or update generic business partners. " +
+                "Updates instead of creating a new business partner if an already existing external ID is used. " +
+                "The same external ID may not occur more than once in a single request. " +
+                "For file upload request, the maximum number of business partners in file limited to \${bpdm.api.upsert-limit} entries.",
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Business partners were successfully updated or created"),
+            ApiResponse(responseCode = "400", description = "On malformed legal entity request", content = [Content()]),
+        ]
+    )
+    @PostMapping("/input/business-partners/uploadCsv", consumes = ["multipart/form-data"])
```

Comment:
Upload Functionality: 
From the discussed requirements (that we should put into an issue somewhere here) it sounds to me that it should be possible to upload also more business partners than just the API limit. This suggests to me, that this endpoint should therefore be an asynchronous operation.
For an asynchronous operation the POST endpoint should immediately return the creation of the upload process job. Also there should be a GET on this resource to get the current status (initial, pending, success, error) as well as (intermediate) results and/or errors.